pkg/vault/test/foundry/Permit2.t.sol

```diff
@@ -136,4 +138,70 @@ contract Permit2Test is BaseVaultTest {
         vm.prank(alice);
         router.permitBatchAndCall(permitBatch, permitSignatures, permit2Batch, bytes(""), multicallData);
     }
+
+    function _setupMulticall()
```

Comment:
Can we add a small `dev` block explaining what the batches do and what the multicalls do?

Reply:
I just simplified the whole thing and got rid of this.